ingest/cli_parser.py

```diff
@@ -331,6 +331,63 @@ def create_parser():
         "--gene-file", help="Path to .genes.tsv file"
     )
 
+    parser_ingest_differential_expression = subparsers.add_parser(
+        "ingest_differential_expression",
+        help="Indicates author differential expression analysis processing",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--ingest-differential-expression",
+        required=True,
+        action="store_true",
+        help="Indicates that author differential expression analysis should be invoked",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--study-accession",
+        required=True,
+        help="Single study accession associated with provided DE input files.",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--annotation-name", required=True, help="Name of annotation for DE analysis"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--annotation-type", required=True, help="Type of annotation for DE analysis"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--annotation-scope", required=True, help="Scope of annotation for DE analysis"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--method", default="wilcoxon", help="method for DE"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--cluster-name", required=True, help="study owner-specified cluster name"
+    )
+
+    # TODO: after end to end support, remove this argument
+    parser_ingest_differential_expression.add_argument(
+        "--cluster-file",
```

Comment:
Blocker: we should remove this and `--annotation-file` since they're not used, and currently not included from the Rails side as arguments.  We would be able to figure them out, but given that they're superfluous it seems preferable to remove them.

Reply:
Done!